src/services/tracing.ts

```diff
@@ -237,21 +231,13 @@ class SpanProcessorWrapper extends SimpleSpanProcessor {
     super.onStart(span, parentContext)
   }
 }
-provider.addSpanProcessor(
-  new SpanProcessorWrapper(
-    new JaegerExporter({
-      host: tracingConfig?.jaegerHost,
-      port: tracingConfig?.jaegerPort,
-    }),
-  ),
-)
+
+provider.addSpanProcessor(new SpanProcessorWrapper(new OTLPTraceExporter()))
```

Comment:
so.. are we trusting in some hidden env variable instead of constructor config? also is intentional to remove config dependency?

Reply:
I guess it's documented here? https://github.com/GaloyMoney/galoy/pull/2812/files#diff-d33e979799a45c7c51752e9c8d96a3e452015d1a40b1e4b6ec6a98e92c4d8430R99

Reply:
it seems to be the recommended way to set up the opel collector https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/

Reply:
I would prefer having explicit code that shows what env vars are being read / injected.
Configuring libraries via env vars that they read out without us explicitly injecting them is really obscure. Isn't there a way to pass the config into the constructor?

Reply:
Constructor accepts this config:
```
export interface OTLPExporterConfigBase {
    headers?: Partial<Record<string, unknown>>;
    hostname?: string;
    url?: string;
    concurrencyLimit?: number;
    /** Maximum time the OTLP exporter will wait for each batch export.
     * The default value is 10000ms. */
    timeoutMillis?: number;
}
```